src/wkt/src/lib.rs

```diff
@@ -31,6 +31,7 @@ mod empty;
 pub use crate::empty::*;
 mod field_mask;
 pub use crate::field_mask::*;
+mod float;
```

Comment:
This will need to be a `pub mod float` sooner or later.

Reply:
I wonder if it should be `internal` since we do not expect application developers to call this API.

Reply:
```suggestion
#[doc(hidden)]
pub mod internal;
```

Reply:
Done.